### ISSUE TITLE
v0.11.0 changes behavior in a breaking way for our app with no attachTo prop is provided

### ISSUE THREAD

**Issue by yacobyonas** [okonet/react-scroll-sync#78]
When we upgraded to version 0.11.0 we noticed that some core app behavior changed and upon digging in, we discovered that it was related to changes in behavior around the `attachTo` prop. If no `attachTo` prop is provided, it will clone the `children` passed in and swap in a new ref ([code](https://github.com/okonet/react-scroll-sync/blob/v0.11.0/src/ScrollSyncPane.js#L91)).

We had code that relied on the fact that the refs would not be swapped out. The culprit PR is #76. Can you advise on how we should be using the API after this change to the behavior? If we want to avoid having `children` be cloned, how should we be utilizing the `attachTo` prop? From reading the code, it wasn't clear how to get the behavior we'd like.

**Comment by okonet** [comment]
Yeah unfortunately I realized later after building this library that using `cloneElement` is very dangerous for exact that reason. 

In your case I would suggest to start using `attachTo` so you can pass the `ref` you need somewhere else to this lib. That's the only way to work around this issue. 

Hope it makes sense.

**Comment by dimitrisnl** [comment]
@yacobyonas I'm on the same boat- would you mind sharing how you overcame this?

**Comment by tearfur** [comment]
Basically the workaround is something like this:

```jsx
// Before
const comp = /* ... */;
return (
  <ScrollSyncPane>{comp}</ScrollSyncPane>
);

// After
const comp = /* ... */;
const compRef = useRef(comp);
return (
  <ScrollSyncPane attachTo={compRef}>{comp}</ScrollSyncPane>
);
```
